### DISCUSSION TITLE
I want to translate it into Chinese

### DISCUSSION THREAD

**Discussion by Genius-pig** [YawLighthouse/UMG-Slate-Compendium#24] [General]
This doc is great. I'm a native Chinese speaker and I have already learned unreal for one year. I want to translate it into Chinese. Already filled out this form.

**Comment by YawLighthouse** [comment]
@Genius-pig Great thank you! I added you to the GitLocalize team, please confirm that you can access it and everything properly.

**Comment by Genius-pig** [comment]
@YawLighthouse Hi, the image link is broken. Should I update every image url manually?


![image](https://github.com/YawLighthouse/UMG-Slate-Compendium/assets/26118649/f47e9a80-0be6-40fc-9162-6a347328b7fe)

**Reply by YawLighthouse** [reply]
Hey, yes if the link is broken then it would need to be fixed. I don't think you have to actually modify the link's themselves, just the caption description's of the image for those using a screen reader and such. I will be frank and honest with you, I've never translated anything before so I would be finding these kinds of errors at the same time you do essentially.

**Reply by YawLighthouse** [reply]
Again thank you so much!!! I really do appreciate it!

**Reply by YawLighthouse** [reply]
If the link is broken on the original english version in GitLocalize but in github it seems fine then you can just ignore it. Apparently its an issue with their system rather than the actual document(which is good I guess)

**Reply by Genius-pig** [reply]
Can I give a pull request in github instead of using gitlocalize bot.

**Reply by YawLighthouse** [reply]
So the pull request through GitLocalize is just how we can track the progress regarding localization or if something became out of date because of a new update or something. So I ask that you please use that.

**Comment by Genius-pig** [comment]
@xalagor Hi, Denis, are you only using GitLocalize web editor to translate the doc? Is there any way to modify the translated file in your own local markdown editor instead of GitLocalize web editor?

**Reply by xalagor** [reply]
Hi. I don't know if there is an option to use another editor. I do almost everything manually in the browser. Check the links in images if GitLocalize tries to use a different folder instead.
![vivaldi_tslhX8jViw](https://github.com/YawLighthouse/UMG-Slate-Compendium/assets/6804367/9fb0669b-b612-489a-830e-c01c887941a5)
